### PR TITLE
New Trauma Physician Uniform

### PR DESCRIPTION
**Pull request by furrycactus**
- Gives Trauma Physicians scrubs and a labcoat, as opposed to that god fucking awful first responder jacket. Their default uniform, not accounting for loadout changes, looks like this now:

https://i.imgur.com/x1fYCfi.png
https://i.imgur.com/5XY7Y9O.png
https://i.imgur.com/smxj2R8.png
https://i.imgur.com/dvHYpka.png

- Also changed a few outstanding red crosses on some species cycled medical voidsuits to green ones.

### CODE REVIEW

**Comment by furrycactus** [comment]
!review

**Comment by alsoandanswer** [comment]
looks way too similar to black scrubs IMHO

also you can upload images direct to github by dragging and dropping, no need to use external hosting like imgur

**Comment by furrycactus** [comment]
I was following the same trend that the surgeon uniform follows; light blue scrubs, light blue labcoat, light blue shoes. I just swapped out the blue for black.

File: code/modules/clothing/shoes/colour.dm [hunk 1]
```diff
@@ -141,6 +141,14 @@
 	permeability_coefficient = 0.01
 	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 90, rad = 0)
 
+/obj/item/clothing/shoes/em_phys
```
Comment (NonQueueingMatt) [inline code/modules/clothing/shoes/colour.dm]: This typepath seems a bit too unwieldy to me personally.

File: code/modules/clothing/suits/labcoat.dm [hunk 2]
```diff
@@ -130,6 +130,14 @@
 	icon_closed = "labcoat_surgeon"
 	armor = list(melee = 0, bullet = 0, laser = 0,energy = 0, bomb = 0, bio = 60, rad = 0)
 
+/obj/item/clothing/suit/storage/toggle/labcoat/em_phys
```
Comment (NonQueueingMatt) [inline code/modules/clothing/suits/labcoat.dm]: Unwieldy typepath. This isn't a blocking issue but it'd be nicer if this was made more sane.

**Comment by furrycactus** [comment]
DNM until I fix the conflicts and space ruins shit that broke.

**Comment by furrycactus** [comment]
Okay it's all good to go now

**Comment by furrycactus** [comment]
I accidentally undid the Trauma Physician name change fuck hang on

**Comment by furrycactus** [comment]
OKAY it's good to go now

**Pull request merged**